local.properties

```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,

```

Comment:
This file should not be in git. .gitignore file should have picked it up, you could check what's wrong. Its not big deal, fix or pass.